src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -987,3 +996,540 @@
+               All domains and origin groups must be connected with a route to enable HTTPS, caching, and custom forwarding logic on specific route paths.
+"""
+
+helps['afd route create'] = """
+type: command
+short-summary: Creates a new route within the specified endpoint.
+examples:
+  - name: Creates a route to assoicate the endpoint's default domain with an origin group for all HTTPS requests.
+    text: >
+        az afd route create -g group --endpoint-name endpoint1 --profile-name profile --route-name route1 --https-redirect False
+        --origin-group og001 --supported-protocols Https --link-to-default-domain Enabled --forwarding-protocol MatchRequest
+  - name: Creates a route to assoicate the endpoint's default domain with an origin group for all requests and use the specified rule sets to customize the route behavior.
+    text: >
+        az afd route create -g group --endpoint-name endpoint1 --profile-name profile --route-name route1 --rule-sets ruleset1 rulseset2
+        --origin-group og001 --supported-protocols Http Https --link-to-default-domain Enabled --forwarding-protocol MatchRequest --https-redirect False
+  - name: Creates a route to assoicate the endpoint's default domain and a custom domain with an origin group for all requests with the specified path patterns and redirect all trafic to use Https.
+    text: >
+        az afd route create -g group --endpoint-name endpoint1 --profile-name profile --route-name route1 --patterns-to-match /test1/* /tes2/*
+        --origin-group og001 --supported-protocols Http Https --custom-domains cd001 --forwarding-protocol MatchRequest --https-redirect True --link-to-default-domain Enabled
+"""
+
+helps['afd route update'] = """
+type: command
+short-summary: Update an existing route within the specified endpoint.
+examples:
+  - name: Update a route to accept both Http and Https requests and redirect all trafic to use Https.
+    text: >
+        az afd route update -g group --endpoint-name endpoint1 --profile-name profile --route-name route1
+        --supported-protocols Http Https --https-redirect True
+  - name: Update a route's rule sets settings to customize the route behavior.
+    text: >
+        az afd route update -g group --endpoint-name endpoint1 --profile-name profile --route-name route1 --rule-sets ruleset1 rulseset2
+  - name: Update a route's compression settings to enable compression for the specified content types.
+    text: >
+        az afd route update -g group --endpoint-name endpoint1 --profile-name profile --route-name route1 --query-string-caching-behavior IgnoreQueryString
+        --enable-compression true --content-types-to-compress text/javascript text/plain
+"""
+
+helps['afd security-policy'] = """
+type: group
+short-summary: Manage security policies within the specified profile. Security policies could be used to apply a web application firewall policy to
```

Comment:
I think it's better to use long-summary here?

Reply:
Agree, thanks a lot.